demisto_sdk/validation/file_validator.py

```diff
@@ -522,18 +531,8 @@ def validate_against_previous_version(self, no_error=False):
             if no_error:
                 self._is_valid = prev_self_valid
 
-    @staticmethod
-    def add_sub_parser(subparsers):
-        parser = subparsers.add_parser('validate', help='Validate your content files')
-        parser.add_argument('-c', '--circle', type=str2bool, default=False, help='Is CircleCi or not')
-        parser.add_argument('-b', '--backward-comp', type=str2bool, default=True,
-                            help='To check backward compatibility.')
-        parser.add_argument('-t', '--test-filter', type=str2bool, default=False,
-                            help='Check that tests are valid.')
-        parser.add_argument('-j', '--conf-json', action='store_true', help='Validate the conf.json file.')
-        parser.add_argument('-i', '--id-set', action='store_true', help='Create the id_set.json file.')
-        parser.add_argument('-p', '--prev-ver', help='Previous branch or SHA1 commit to run checks against.')
-        parser.add_argument('-g', '--use-git', action='store_true', help='Validate changes using git.')
+    # parser.add_argument('-t', '--test-filter', type=str2bool, default=False,
```

Comment:
TODO.

after validation there was a step to run the `configure_tests` script to check that each changed file had a relevant test - was used as part of the hooks.